src/vr/Touchable.js

```diff
@@ -146,35 +141,31 @@ const Touchable = (
     },
 
     setPressValue(toValue) {
-      Animated.timing(
-        this.props.press,
-        {
-          toValue,
-          duration: this.props.pressDuration,
-          // easing: Easing.inOut(Easing.quad),
-        }
-      ).start();
+      Animated.timing(this.props.press, {
+        toValue,
+        duration: this.props.pressDuration,
+        // easing: Easing.inOut(Easing.quad),
+      }).start();
     },
 
-
-    touchableHandleActivePressIn: throttle(function (e) {
+    touchableHandleActivePressIn: throttle(function(e) {
```

Comment:
definitely please revert this :-) `function`, space, optional name, argument list, always